amd/build/comment_area.min.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=comment_area.min.js.map
```

Comment:
https://travis-ci.org/frankkoch/moodle-mod_studentquiz/jobs/639434385#L1013. Missing build or is it this bug? https://github.com/blackboard-open-source/moodle-plugin-ci/issues/109 (I don't see the other files appear)